master/static/migrations/20220829162026_set-raw-metrics-archive.tx.up.sql

```diff
@@ -0,0 +1,16 @@
+UPDATE raw_steps update_steps
+SET archived = TRUE
+FROM ( -- Get max trial_run_id for each (trial_id, total_batches) pair
+    SELECT trial_id, total_batches, max(trial_run_id) AS trial_run_id FROM raw_steps
```

Comment:
i am slightly worried about the cost of this migration. can you dry run against latest master and get a plan/time estimate. the only thing worrying is the window over checkpoints view.

Reply:
if it is too expensive, we can just warn in the release note and not do it.

Reply:
Dry run takes ~500ms on latest master with ~4000 checkpoints in the ```checkpoints_view```

Reply:
wonder when there are also a lot of metrics/vals since the checkpoints view.. sucks. but i'll take this i tihnk.